PhysicsTools/PatUtils/python/tools/runMETCorrectionsAndUncertainties.py

```diff
@@ -1443,8 +1448,10 @@ def ak4JetReclustering(self,process, pfCandCollection, patMetModuleSequence, pos
             pfCHS=None
             if self._parameters["onMiniAOD"].value: 
                 pfCHS = cms.EDFilter("CandPtrSelector", src = pfCandCollection, cut = cms.string("fromPV"))
-                setattr(process,"pfNoPileUpJME"+postfix,pfCHS)
+                #setattr(process,"pfNoPileUpJME"+postfix,pfCHS)
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant

Reply:
@slava77  No is not - sorry. We fixed it with a new commit.